modules/govuk_elasticsearch/manifests/init.pp

```diff
@@ -89,6 +98,12 @@
         },
       },
     },
+    'cloud'                    => {
+      'aws' => {
+        'access_key' => $aws_access_key,
+        'secret_key' => $aws_secret_key,
+      },
+    },
```

Comment:
I think this is fine to pass into config, assuming that only the elasticsearch user has access to it.